### PR TITLE
Update: CWS readme procedure

### PR DESCRIPTION
**Pull request by voxparcxls**
#### Added to README
* Installing Docker prerequisite.
* Described Elasticsearch 3 form configuration: HTTPS with Authen, HTTPS without Authen, HTTP.
* Described running Elasticsearch with Docker container. (alternative to installing ES)
* Fixed typos.

### CODE REVIEW

File: README.md
Comment (jamesfwood) [inline README.md:106]: Also update **_personal_dev.sh_** script to include the "http://"

Comment (jamesfwood) [reply README.md:106]: Never mind.  Leave it like this.

**Review by jamesfwood — APPROVED** [review]

**Comment by jamesfwood** [comment]
I will merge this after we merge the ES w/o auth ticket.

**Review by ztaylor54 — APPROVED** [review]
Looks good, thanks @voxparcxls!

**Pull request merged**
